backend/repositories/ContentPublisherRepository.ts

```diff
@@ -0,0 +1,64 @@
+import { Client as ContentPublisherClient } from "../types/openapi-content-publisher.js";
+import type { Components, Paths } from "../types/openapi-content-publisher.js";
+import OpenAPIClientAxios from "openapi-client-axios";
+export class ContentPublisherRepository {
+  private static instance: ContentPublisherRepository;
+  private _client: ContentPublisherClient;
```

Comment:
whats the difference between _client and client? Purpose of "_"?

Reply:
To distinguish between the getter and setter functions
```
  private set client(api: ContentPublisherClient) {
    this._client = api;
  }

  private get client() {
    if (this._client === undefined) {
      throw new Error("API not initialized");
    }

    return this._client;
  }
  ```